src/nni_manager/common/utils.ts

```diff
@@ -33,7 +33,7 @@ import * as util from 'util';
 
 import { Database, DataStore } from './datastore';
 import { ExperimentStartupInfo, getExperimentId, getExperimentStartupInfo, setExperimentStartupInfo } from './experimentStartupInfo';
-import { Manager } from './manager';
+import { Manager, ExperimentStartUpMode } from './manager';
```

Comment:
Is this import necessary?  it seems not used.

Reply:
removed.